internal/metrics/prometheus.go

```diff
@@ -152,6 +154,23 @@ func NewCtrlFuncMetrics() *CtrlFuncMetrics {
 		[]string{SuccessKey},
 	)
 
+	// TODO: add new metric for fallback config generation or add a "fallback" label?
+	controllerMetrics.TranslationDuration = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name: MetricNameTranslationDuration,
+			Help: fmt.Sprintf(
+				"Duration of translations from Kubernetes state to Kong state."+
+					"`%s` describes whether there were unrecoverable errors (`%s`) or not (`%s`). "+
+					"Unrecoverable error in this case means KIC wasn't able to translate a Kubernetes object to Kong model.",
+				SuccessKey, SuccessFalse, SuccessTrue,
+			),
+			// TODO: A simple configuration translation with 1 ingress and 1 service costs 100~200 microseconds.
+			// Should we use a smaller starting value of the buckets?
+			Buckets: prometheus.ExponentialBuckets(1, 2, 20),
```

Comment:
To align with other duration metric bucket count in this file.

```suggestion
			Buckets: prometheus.ExponentialBuckets(1, 2, 30),
```

Reply:
`2^19` ms ~ 500s Should be large enough, while `2^29` ms ~ 500000s is nearly a week. If we want to keep the number of the buckets to 30, I think using the `ExponentialBucketsRange` to specify the interval is clearer (about 1 ms ~ 1 hour)?

Reply:
Ok fair point, we can use the originally proposed buckets.

Reply:
Changed to `ExponentialBucketsRange(1, 60*60*1000, 30)`. Which one do you think is more suitable for this bucket, the original one or the new one?

Reply:
This one makes a lot of sense 👍 

I believe we can use the time consts for this to make it more expressive:

```
prometheus.ExponentialBucketsRange(1, float64((30*time.Second)/time.Millisecond), 30)
```

Since 30s is probably going to be more than enough and timeouts will kick I think this is a reasonable max. We can also make a note about this in the comment. WDYT?